config/overlays/tilt-local-dev/operator-controller/patches/dev-deployment.yaml

```diff
@@ -0,0 +1,13 @@
+# remove livenessProbe and readinessProbe so container doesn't restart during breakpoints
+- op: replace
+  path: /spec/template/spec/containers/0/livenessProbe
+  value: null
+- op: replace
+  path: /spec/template/spec/containers/0/readinessProbe
+  value: null
+- op: remove
+  # remove --leader-elect so container doesn't restart during breakpoints
+  path: /spec/template/spec/containers/0/args/2
+- op: add
+  path: /spec/template/spec/containers/0/args/-
+  value: --feature-gates=PreflightPermissions=true
```

Comment:
This should probably be removed from the general tilt-dev overlay. But I imagine we should make a follow-up issue (if we don't have one already) for making it easy for contributors to enable non-default feature gates.

Reply:
So, keep or remove?

Reply:
Remove, IMO. But let's make a follow-up to decide to we want to turn feature gates on and off easily. I'm guessing @trgeiger added this, so let's see what his thoughts are.

Reply:
Removed, but I don't want this to hold things up...

Reply:
@tmshort Can you create an issue to track this issue i.e. Adding a way for enabling non-default feature gates in tilt?

Reply:
Yeah I'm fine with removing, I wasn't even sure about it when I added it. It's simple enough for devs to add any flags they need locally.

Reply:
Created #1793